adapters/avocet/avocet.go

```diff
@@ -0,0 +1,120 @@
+package avocet
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// AvocetAdapter implements a adapters.Bidder compatible with the Avocet advertising platform.
+type AvocetAdapter struct {
+	// Endpoint is a http endpoint to use when making requests to the Avocet advertising platform.
+	Endpoint string
+}
+
+func (a *AvocetAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	headers := http.Header{}
```

Comment:
There's a lovely elegancy to just passing through the OpenRTB request. It's common for adapters to do a bit of checking before making the network call to place a bid, aka.. to fail fast. This typically includes checking for no impressions and checking for a compatible ad type. It's also common to pull out the W and H of banner ads from the formats slice.

Do you think any of this applies to the Avocet adapter?